inc/define.php

```diff
@@ -254,7 +254,7 @@
     'SoftwareLicense', 'Certificate', 'Appliance', 'Item_DeviceSimcard', 'Line'
 ];
 
-$CFG_GLPI["linkgroup_types"]              = ['Computer', 'Consumable', 'Monitor', 'NetworkEquipment',
+$CFG_GLPI["linkgroup_types"]              = ['Computer', 'Monitor', 'NetworkEquipment',
```

Comment:
Could you explain this change?

Reply:
You should not remove this item from this list. It can be linked to a group, so it has to be defined here. Some plugins are using this list and removing this class from the list may break some features.

Instead, you should either add this field to `Consumable` (I do not know this part, so I do not know if it can make sense), either adapt your query using this kind of code:
```php
if ($item->isField('users_id')) {
    ...
}
```

Reply:
My bad, there is no `group_id` field. List was probably wrong.